scripts/servers.py

```diff
@@ -638,6 +639,11 @@ def managed_protractor_server(
     if sharding_instances <= 0:
         raise ValueError('Sharding instance should be larger than 0')
 
+    if mobile:
+        os.environ['MOBILE'] = 'true'
+    else:
+        os.environ['MOBILE'] = 'false'
```

Comment:
Please make sure these get reverted after the tests complete, even if the tests fail partway through (the standard way to do this is with a context manager). We don't want to permanently change the user's environment to avoid causing side-effects

Reply:
As the parent function already uses the `contextlib.contextmanager` decorator, I have added the following statement to handle the environment variable changes: 
![image](https://user-images.githubusercontent.com/73544247/190328956-81c9bf18-af61-4238-b540-5de76d3d74ef.png)
Is it the correct way to do it?